app/src/pages/react-uswds-hello.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  Alert,
+  Button,
+  Grid,
+  GridContainer,
+  Icon,
+} from "@trussworks/react-uswds";
+import Link from "next/link";
+import React from "react";
+
+/** Example page showing use of react-uswds components. Can be deleted once you're on a roll. */
+const HelloWorld = () => {
```

Comment:
non-blocking scope creep suggestion: could we add a form example to this? I think a lot of our projects revolve around forms, so could be helpful to have a reference when first starting with `react-uswds`